src/pages/Home.jsx

```diff
@@ -38,6 +40,20 @@ const Home = () => {
       >
         <section id="home">
           <Stack alignItems="center" justifyContent="center">
+            <Box my={8}>
+              <Box
+                component="img"
+                src="/assets/PASSLogolightmode.png"
+                alt=""
```

Comment:
There are a few empty `alt`s like here and `componentImageAlt=""` below.

Reply:
Yeah, this should have something like "PASS logo." as the value.

Reply:
@milofultz 
Ok, I was following the info that was all discussed/linked previously here: https://github.com/codeforpdx/PASS/pull/458

But maybe it's different for the logo?

@andycwilliams 
What I linked above is the discussion on all the empty alts for the other images that were already on this page (the alts were previous PR, I didn't change them.)

Reply:
Yeah logos are special. TBH I still don't quite grok the why, but every space I check says you should add an alt attribute for the logo. I think you are generally correct that this is decorative, but seems like logos have a special case